docs/spec/spec.md

```diff
@@ -2138,15 +2138,23 @@ public isolated class RequestContext {
     # + return - Attribute value or an error if the member value is not of the expected type
     public isolated function getWithType(string key, ReqCtxMemberType targetType = <>) returns targetType|ListenerError = external;
 
-    # Removes an member from the request context object. It panics if there is no such member.
+    # Removes a member from the request context object. It panics if there is no such member.
     #
     # + key - Represents the member key
     public isolated function remove(string key) {}
+    
+    # Provides the JWT information from the request.
+    #
+    # + return - `[jwt:Header, jwt:Payload]` if decoding the header is successful, `error` if any error occurs
+    #             while decoding, or `nil` if no jwt header found.
+    public function getJWTInfo() returns [jwt:Header, jwt:Payload]|error? {}
 
     # Calls the next service in the interceptor pipeline.
     #
     # + return - The next service object in the pipeline. An error is returned, if the call fails
     public isolated function next() returns NextService|error? = external;
+
+    isolated function getAuthString() returns string|error? = @java:Method {}
```

Comment:
This we don't need to expose through spec since the function is not public.

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-http/pull/1470/commits/461c0635c2a83486b45f6fa21243bf23fbeb8227